src/messageHandler.ts

```diff
@@ -180,6 +203,20 @@ class MessageHandler {
     if (this.callbacks.onShare) await this.callbacks.onShare(event);
   };
 
+  private onTransferConsentRequested = async (event: TransferConsentRequestedEvent) => {
+    console.log('SDK onTransferConsentRequested', this.callbacks.onTransferConsentRequested);
```

Comment:
it should stay? :D

Reply:
Nope. (a good [movie](https://www.imdb.com/title/tt10954984/) btw)